site/hxapi/asynchronous-bookings.md

```diff
@@ -0,0 +1,143 @@
+---
+
+---
+
+# Asynchronous Bookings
+
+## Background
+
+We now offer products that use an asynchronous booking process. This allows partners to handle bookings for a wider range of products. However, asynchronous bookings require a different approach compared to synchronous bookings to provide a seamless experience for customers.
+This guide outlines:  
+- Best practices for integrating with our API for asynchronous bookings.  
+- Guidance on when and how to poll our API.  
+- Details on booking statuses.  
+- Information on enabling availability for async products.  
+
+---
+
+## Enabling Async Product Availability  
+
+To view all supply, including async products, partners can now specify whether they want these products included in the availability response.
+
+### Availability Request  
+
+Partners must include a new parameter in the availability request to indicate whether to include async products.  
+
+| Name                    | Data Type | Format  | Mandatory? | Additional Information                                    |
+|-------------------------|-----------|---------|------------|----------------------------------------------------------|
+| includeAsyncProducts    | Boolean   | `true`/`false` | N | If `true`, availability includes async products. Default is `false`. |
+
+### UK Products Availability Request
+
+{% codetabs %}
+{% codetab XML %}
+```
+https://api.holidayextras.co.uk/carpark/LGW?ABTANumber=YourABTANumber&Password=YourPassword&Initials=YourInitials&key=YourKey&token=YourToken&ArrivalDate=2020-01-07&ArrivalTime=1200&DepartDate=2020-01-08&DepartTime=1200&NumberOfPax=1&includeAsyncProducts=true
+```
+{% endcodetab %}
+{% codetab JSON %}
+```
+https://api.holidayextras.co.uk/carpark/LGW.js?ABTANumber=YourABTANumber&Password=YourPassword&Initials=YourInitials&key=YourKey&token=YourToken&ArrivalDate=2020-01-07&ArrivalTime=1200&DepartDate=2020-01-08&DepartTime=1200&NumberOfPax=1&includeAsyncProducts=true
+```
+Note the `.js` extension.
+{% endcodetab %}
+{% endcodetabs %}
+
+### European Products Availability Request
+
+{% codetabs %}
+{% codetab XML %}
+```
+https://api.holidayextras.co.uk/carpark/MUC?ABTANumber=YourABTANumber&Password=YourPassword&Initials=YourInitials&key=YourKey&token=YourToken&ArrivalDate=2017-12-01&ArrivalTime=1200&DepartDate=2017-12-08&DepartTime=1200&NumberOfPax=1&System=ABG&lang=en&includeAsyncProducts=true
+```
+{% endcodetab %}
+{% codetab JSON %}
+```
+https://api.holidayextras.co.uk/carpark/MUC.js?ABTANumber=YourABTANumber&Password=YourPassword&Initials=YourInitials&key=YourKey&token=YourToken&ArrivalDate=2017-12-01&ArrivalTime=1200&DepartDate=2017-12-08&DepartTime=1200&NumberOfPax=1&System=ABG&lang=en&includeAsyncProducts=true
+```
+Note the `.js` extension.
+{% endcodetab %}
+{% endcodetabs %}
+
+
+### Behavior  
+
+- **`true`:** Availability searches include async products.  
+- **`false`:** Availability searches exclude async products (default behavior).  
+
+---
+
```

Comment:
As a partner, I think I'd be asking the following questions at this point:
- "Will I be able to see in the availability response that specific products are asynchronous?"
- "Do I need to book async products differently from traditional synchronous ones? Or must all products now be booked using this new approach? What changes do I need to make to the actual Booking call?"
- "Will the response to the Booking call change at all? What will it look like for the booking of an async product?"

Example booking requests & responses will go a long way in explaining this.

Reply:
Adjusted 👍